### ISSUE TITLE
Joint training SISDR and Pesq loss

### ISSUE THREAD

**Issue by Niteshkumarchaudhary** [audiolabs/torch-pesq#8]
Hi, what should be the alpha value for the joint SISDR and pesq loss training (SISDR_loss + alpha*pesq_loss) ?
Firstly I checked the range of each loss (SISDR and pesq) then scaled them in the same range and used alpha = 0.5 (equal weightage) then what I observed that after training, in processed output significant amount of speech signal also gets suppressed. Please comment and let me know.

I have another question for the pesq loss below
pesq = PesqLoss(val,
    sample_rate=44100, 
)
what is val in PesqLoss ? is it just for scaling the loss value or something else ?

**Comment by bytesnake** [comment]
Hi! Thank you for your interest.

> I have another question for the pesq loss below pesq = PesqLoss(val, sample_rate=44100, ) what is val in PesqLoss ? is it just for scaling the loss value or something else ?

The value in that context is the loss weight. In the experiments, SI-SDR and PESQ had same weighting (lamb=1/2). 

> [..] in processed output significant amount of speech signal also gets suppressed. Please comment and let me know.

that depends on your model architecture, I can't comment on the performance without knowing the architecture. For our model the improvement was non-neglectable but minor. Maybe you can perform a hyperparameter search for your loss weights to find a good balance.

**Comment by bytesnake** [comment]
For further help, please reopen the issue
